Automattic/jetpack
# Replace /plans and /plans-prompt routes with a redirect to cloud.jetpack.com/pricing

Fixes 1164141197617539-as-1191506083092569

#### Changes proposed in this Pull Request:

* Replace the `/plans` and `/plans-prompt` pages with a redirect to the new pricing page located in `cloud.jetpack.com/pricing`, while keeping the current site in context (carried over as a query parameter).

#### Jetpack product discussion

You can read the motivation behind this change in the ticket mentioned above. In short, we shouldn't allow users to visit the Plans page because we stopped selling those plans after the Offer Reset project launch.

#### Does this pull request change what data or activity we track or use?

I don't think this PR changes anything related to tracks.

#### Testing instructions:






Prerequisite: a Jetpack site.

* Run this PR (locally or with the Beta Tester plugin).
* Visit `:siteUrl/wp-admin/admin.php?page=jetpack#/plans` replacing `siteUrl` with your real site URL.
* Verify that you are redirected to `https://cloud.jetpack.com/pricing/?site=:siteSlug&source=jetpack-plans`.
* Visit `:siteUrl/wp-admin/admin.php?page=jetpack#/plans-prompt` replacing `siteUrl` with your real site URL.
* Verify that you are redirected to `https://cloud.jetpack.com/pricing/?site=:siteSlug&source=jetpack-plans`.

#### Visiting `/plans`
![JetpackPlans](https://user-images.githubusercontent.com/3418513/98845199-200fcf00-242c-11eb-9da2-46fb4326cb2a.gif)

#### Visiting `/plans-prompt`
![JetpackPlansPrompt](https://user-images.githubusercontent.com/3418513/98845193-1dad7500-242c-11eb-83a7-44cc628e2ca5.gif)


#### Proposed changelog entry for your changes:

* Replace /plans and /plans-prompt routes with a redirect to cloud.jetpack.com/pricing.
